src/assets/i18n/en.json

```diff
@@ -1,4 +1,6 @@
 {
+	"Managed by Arlas": "This field will be automatically filled by Arlas",
+	"Field set by unit collection in look and feel": "This field is automatically filled in with the collection unit entered in look and feel. If you want to change this value, go to the look and feel tab.",
```

Comment:
I propose : 
"In case of displaying the count on the y-axis, the unit of the displayed count corresponds to the chosen collection unit. Therefore; in order to change this unit, you need to change the collection unit in the 'Look 'n feel' section.".